front/src/pages/Admin/User/ListUsers/ListUsers.tsx

```diff
@@ -46,14 +161,35 @@ const Page = declareAdminRoute(function ListUsers() {
   }
 
   const users = data!.User.list;
+  const filteredUsers = filterUsers(users, filters);
+  const previousUrl = location.pathname + location.search;
+  // todo: hard delete on second user deletion with a red warning.
 
-  return <>
-    <h1>Liste des utilisateurs, admin</h1>
-    {users.map(user => {
-      return <p key={user.uid}>{user.email}</p>;
-    })}
-  </>;
-}, ROUTE_LIST_USERS);
+  return <Box p="md">
+    {uidDeleted && <Alert title="Success" variant="success"> {uidDeleted} : {trans('pages.admin.user.form.success')}</Alert>}
+    {mappedErrors.__root && <Alert title="Error" variant="danger">{mappedErrors.__root}</Alert>}
+    <Group>
+      <Title order={1}>{trans('pages.admin.user.list.documentTitle')}</Title>
+      <Button>
+        <Link
+          style={{
+            color: 'white',
+          }} to={route(UserCreate)}
+        >{trans('pages.admin.user.create.button')}</Link>
```

Comment:
Polymorphic component like above:
```suggestion
      <Button
        component={Link}
        to={route(UserCreate)}
      >
        {trans('pages.admin.user.create.button')}
```